oceanbasecloud/metadata.csv

```diff
@@ -0,0 +1,16 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+oceanbasecloud.load1,gauge,,,,Past 1 minute system average load,1,oceanbasecloud,,
+oceanbasecloud.cpu_percent,gauge,,percent,,CPU Percent,1,oceanbasecloud,,
```

Comment:
Could you please clarify whether this is the percentage of CPU being _utilized_ or _free_?

Reply:
this is the percentage of CPU being utilized